www/js/app.js

```diff
@@ -117,7 +117,14 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             }, 100);
         }
     }
-    $(document).ready(resizeIFrame);
+    $(document).ready(function(){
+        resizeIFrame();
+        //handle home key press in initial Home Page (empty iframe)
+        var iframeContentWindow = document.getElementById('articleContent').contentWindow;
+        $(iframeContentWindow).on('keydown',function (e){
```

Comment:
Why are you using jQuery here? You have defined `iframeContentWindow` using native DOM methods, so jQuery is redundant and just adds overhead. You simply want `iframeContentWindow.addEventListener('keydown', function ...`

Reply:
Thank you for indicating. Changes made in new commit.